worker/xml/tasks.py

```diff
@@ -0,0 +1,15 @@
+from worker.tasks import BaseTask
+from worker.xml.xml_generator import generate_xml
+from utils.celery_client import celery_app
+
+
+@celery_app.task(bind=True, name="generate_marcxml", base=BaseTask)
+def task_generate_marcxml(self, object_id, job_id):
```

Comment:
Vorschlag: Auf einen generischeren Task `task_generate_xml` reduzieren und das Template als Parameter in der job_config.yml setzen.